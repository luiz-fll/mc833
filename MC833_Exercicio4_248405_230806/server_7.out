remoto: 127.0.0.1:6007
[SERVIDOR] Escutando em 127.0.0.1:6007
127.0.0.1:54734
GET / HTTP/1.0
Host: teste

127.0.0.1:54736
GET / HTTP/1.0
Host: teste

127.0.0.1:54738
GET / HTTP/1.0
Host: teste

127.0.0.1:54746
GET / HTTP/1.0
Host: teste

127.0.0.1:54762
GET / HTTP/1.0
Host: teste

127.0.0.1:54772
GET / HTTP/1.0
Host: teste

127.0.0.1:54786
GET / HTTP/1.0
Host: teste

127.0.0.1:54794
GET / HTTP/1.0
Host: teste

127.0.0.1:54804
GET / HTTP/1.0
Host: teste

