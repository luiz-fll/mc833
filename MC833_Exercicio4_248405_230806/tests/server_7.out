remoto: 127.0.0.1:60007
[SERVIDOR] Escutando em 127.0.0.1:60007
127.0.0.1:55268
GET / HTTP/1.0
Host: teste

127.0.0.1:55270
GET / HTTP/1.0
Host: teste

127.0.0.1:55278
GET / HTTP/1.0
Host: teste

127.0.0.1:55290
GET / HTTP/1.0
Host: teste

127.0.0.1:55302
GET / HTTP/1.0
Host: teste

127.0.0.1:55306
GET / HTTP/1.0
Host: teste

127.0.0.1:55322
GET / HTTP/1.0
Host: teste

127.0.0.1:55328
GET / HTTP/1.0
Host: teste

127.0.0.1:55332
GET / HTTP/1.0
Host: teste

